runtime/common/src/extensions/check_nonce.rs

```diff
@@ -99,7 +99,7 @@ where
 		}
 
 		// Increment account nonce by 1
-		account.nonce += T::Index::one();
+		account.nonce += T::Nonce::one();
```

Comment:
nit: unsafe increment but I'm not sure if `T::Nonce` supports any safe ones

Reply:
agreed something like `saturating_add`

Reply:
I will create an issue for this since its outside the scope of the PR.

Reply:
https://github.com/LibertyDSNP/frequency/issues/1724

Reply:
Actually I'm not sure if it is an issue if the nonce rolls over. We might need it to even.